conftest.py

```diff
@@ -0,0 +1,15 @@
+"""Configure pytest"""
```

Comment:
it looks like this fixture is only needed in the `exams` app. if that's right, can you move this `conftest.py` file to that directory?